<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKeyboardForm</class>
 <widget class="QWidget" name="CKeyboardForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>-1</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QSlider" name="pitchSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:vertical 
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(50, 50, 50, 255));
	width: 10px;
	border: 1px inset #555;
	border-radius: 3px;
}

QSlider::handle:vertical 
{
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #222, stop:1 #ddd);
	border: 1px solid #888;
	height: 6px;
	/*margin-left: -2px;
	margin-right: -2px;*/
	border-radius: 3px;
}
</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="modulationSlider1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:vertical 
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(50, 50, 50, 255));
	width: 10px;
	border: 1px inset #555;
	border-radius: 3px;
}

QSlider::handle:vertical 
{
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #222, stop:1 #ddd);
	border: 1px solid #888;
	height: 6px;
	/*margin-left: -2px;
	margin-right: -2px;*/
	border-radius: 3px;
}
</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="modulationSlider2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:vertical 
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(50, 50, 50, 255));
	width: 10px;
	border: 1px inset #555;
	border-radius: 3px;
}

QSlider::handle:vertical 
{
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #222, stop:1 #ddd);
	border: 1px solid #888;
	height: 6px;
	/*margin-left: -2px;
	margin-right: -2px;*/
	border-radius: 3px;
}
</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="OCPiano" name="piano" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OCPiano</class>
   <extends>QWidget</extends>
   <header location="global">ocpiano.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
